pylana/logs.py

```diff
@@ -13,6 +13,8 @@
 from .utils import create_case_semantics_from_df, create_event_semantics_from_df
 from .decorators import expect_json
 from .decorators import handle_response
+from pathlib import Path
```

Comment:
[PEP 8](https://www.python.org/dev/peps/pep-0008/) suggests to import standard libraries first.